src/test/java/org/jenkinsci/plugins/registry/notification/webhook/WebHookPayloadTest.java

```diff
@@ -43,7 +43,8 @@ public void testSerialization() throws Exception {
         json.put("repository", repository);
         WebHookPayload obj = new DockerHubWebHookPayload(json);
 
-        XStream xs = new XStream();
+        XStream2 xs = new XStream2();
+        xs.allowTypes(new Class[] {DockerHubWebHookPayload.class});
```

Comment:
Not really necessary in the case of `Xstream2`, but I left it in since that is technically the new way going forward.